### ISSUE TITLE
Outlook 2010: Filename of Attachment Not Announced after NVDA Restart

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#7528]
STR:

1. Compose a new message, and attach any file

2. From within the message window, navigate to the "Attachment" entry; the filename and size of the attached file will be announced

3. Restart NVDA, and navigate back to the open message

Expected: When navigating back to the attachment entry, the filename and size should be output as previously

Actual: The text "Outlook File Attachment" is announced; alt-tabbing out of and back into Outlook resolves this, but not until the above steps are carried out

**Comment by Adriani90** [comment]
@elliott94 are you still able to reproduce this issue in Outlok 2010? I have Office 2010 installed but Outlook seems to be broken on my end so a test from my side would not be reliable just now.

**Comment by lukaszgo1** [comment]
I can reproduce it with Outlook 2010 32-bit installed on Windows 7 x64 with latest NVDA Alpha.

**Comment by Adriani90** [comment]
@lukaszgo1 I cannot reproduce it in Outlook 2016 or newer with NVDA 2023.1. Can you confirm? Then I would propose to close this as fixed since support for Office 2010 ended completely in 2020, and end of live for Office 2013 is in April 2023, so I don't expect any further updates to these Office versions.

**Comment by Adriani90** [comment]
No updates from the initial author or anyone else who can reproduce this on Office versions 2016 and newer. I cannot reproduce in Office 2016 and newer. Closing as works for me. If anyone still has this issue, please comment and we can reopen.
